content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+Our biggest release just yet
+
+<!--more-->
+
+Hello Jellyfin users out there! Welcome back to our blog!
+
+Fasten your seat belts and get ready because the numbers are big: Since 10.4.3, **over 200 contributions** from the community were merged
+and **over 500 issues** were fixed**Over 200 contributions from the community were merged and over 500 issues were fixed** both in Jellyfin and Jellyfin Web, the interface you access using your browser.
+
+We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
+We hope you will forgive us after trying it (*and maybe send us some cakes*)!
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We sure hope we have!. Anyway, you're advised to take some popcorn before continuing. Grab some popcorn and get ready to dive in!
```

Comment:
Here's a rephrasing that I tried to keep close to the tone of the original but I think flows a little bit better:

> Although we tried hard, there's so much packed into this release that we are still unsure if this blog post actually manages to cover all the awesome new features. We sure hope it does! Either way, there's a lot of content to cover so grab some popcorn and get ready to dive in!

Reply:
I really like this one, thank you!